doc/README-pt-br.md

```diff
@@ -36,30 +36,30 @@ Não esqueça o `enctype="multipart/form-data"` em seu formulário.
 
 ```javascript
 const express = require('express')
-const multer  = require('multer')
+const multer = require('multer')
 const upload = multer({ dest: 'uploads/' })
 
 const app = express()
 
 app.post('/profile', upload.single('avatar'), function (req, res, next) {
-  // req.file is the `avatar` file
-  // req.body will hold the text fields, if there were any
+  // req.file é um arquivo `avatar`
+  // req.body conterá os campos de texto, se houver
 })
 
 app.post('/photos/upload', upload.array('photos', 12), function (req, res, next) {
-  // req.files is array of `photos` files
-  // req.body will contain the text fields, if there were any
+  // req.files é um array de arquivos `photos`
+  // req.body conterá os campos de texto, se houver
 })
 
 const cpUpload = upload.fields([{ name: 'avatar', maxCount: 1 }, { name: 'gallery', maxCount: 8 }])
 app.post('/cool-profile', cpUpload, function (req, res, next) {
-  // req.files is an object (String -> Array) where fieldname is the key, and the value is array of files
+  // req.files é um objeto (String -> Array) onde fieldname é a chave e o valor é array de arquivos
   //
   // e.g.
-  //  req.files['avatar'][0] -> File
+  //  req.files['avatar'][0] -> Arquivo
```

Comment:
This is referencing the class `File`, so I think that it should probably stay as File?

```suggestion
  //  req.files['avatar'][0] -> File
```